addOns/pscanrulesBeta/src/test/java/org/zaproxy/zap/extension/pscanrulesBeta/CacheableScanRuleUnitTest.java

```diff
@@ -131,6 +133,37 @@ void shouldNotCauseExceptionWhenExpiresHeaderHasZeroValue()
         assertThat(alertsRaised.size(), equalTo(1));
     }
 
+    @ParameterizedTest
+    @ValueSource(
+            strings = {
+                "s-maxage=600,max-age=0",
+                "s-maxage=600, max-age=0",
+                "s-maxage=600",
+                "s-maxage=600,",
+                "max-age=0,s-maxage=600"
+            })
+    void shouldHandleVariousMaxAgeFormats(String headerValue)
+            throws URIException, HttpMalformedHeaderException {
+        // Given
+        HttpMessage msg = createMessage();
+        msg.setResponseHeader(
+                "HTTP/1.1 200 OK\r\n"
+                        + "Cache-Control: "
+                        + headerValue
+                        + "\r\n"
+                        + "Pragma: must-revalidate,no-cache\r\n"
+                        + "Content-Type: text/xml;charset=UTF-8\r\n"
+                        + "Expires: 0\r\n"
+                        + // http-date expected, Ex: "Wed, 21 Oct 2015 07:28:00 GMT"
```

Comment:
Redundant, that a method is being called already implies that a certain date/format is needed. (Can be removed from the origin too.)